{{ cookiecutter.project_slug }}/backend/mx.ini

```
@@ -0,0 +1,15 @@
+; This is a mxdev configuration file
+; it can be used to override versions of packages already defined in the
+; constraints files and to add new packages from VCS like git.
+; to learn more about mxdev visit https://pypi.org/project/mxdev/
+
+[settings]
```

```suggestion
[settings]
default-target = src
```

cookiestarter-plone-starter generates policy package in /src so any other package checked out for development could follow in /src and not in /sources like default mxdev does.

---

hm. I would like to keep sources, because src can be confused with the src inside a python package.